VoiceAttackResponder/VoiceAttackPlugin.cs

```diff
@@ -675,9 +675,11 @@ public static void InvokeSay(ref dynamic vaProxy)
 
                 string voice = vaProxy.GetText("Voice");
 
+                int? volume = vaProxy.GetInt("Volume") ?? 100;
+
                 string speech = SpeechFromScript(script);
 
-                SpeechService.Instance.Say(((ShipMonitor)EDDI.Instance.ObtainMonitor("Ship monitor")).GetCurrentShip(), speech, (int)priority, voice, false, null, true);
+                SpeechService.Instance.Say(((ShipMonitor)EDDI.Instance.ObtainMonitor("Ship monitor")).GetCurrentShip(), speech, (int)priority, voice, false, null, true, (int)volume);
```

Comment:
If `transmitVolume` was not dependent on `Configuration.Volume`, there would be no need to pass an override volume until we needed to use it (the .ctor could default to null, as I proposed above).